src/utils/constants.ts

```diff
@@ -0,0 +1,6 @@
+import { BigInt } from '@graphprotocol/graph-ts'
+
+export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
+export const ONE_ETHER = BigInt.fromString('1000000000000000000')
+export const ONE_HUNDRED_ETHER = BigInt.fromString('100000000000000000000')
+export const MAX_EXPIRATION_DATE = BigInt.fromString('9999999999')
```

Comment:
`MAX_EXPIRATION_DATE` is part of orium business logic in other subgraph. Since we are going for generic approach. We can remove this for now. 

PS: In future, that number would be probably max uint256 which is a huge number!